content/docs/rfcs/57/README.md

```diff
@@ -287,16 +287,88 @@ This allows the Status app to abstract from the discovery process and simply add
 
 # DoS Protection
 
-> *Note* :  DoS protection will be specified in a soon-to-follow update of this RFC (while in raw state).
-The following sketches basic approaches.
-DoS protection might move in a separate document and be referenced here.
+Hereunder we describe the "opt-in message signing for DoS prevention" solution, designed *ad hoc* for Status MVP.
 
-## Statically-Mapped Communities
+Since publishing messages to pubsub topics has no limits, anyone can publish messages at a very high rate and DoS the network.
+This would elevate the bandwidth consumption of all nodes subscribed to said pubsub topic, making it prohibitive (in terms of bandwidth) to be subscribed to it.
+In order to scale, we need some mechanism to prevent this from happening, otherwise all scaling efforts will be in vain.
```

Comment:
Not sure if worth specifying that other scaling efforts address splitting traffic and reducing duplicate messages so it's clearer why it'd be vain.